src/app/views/home/sections/Landing.scss

```diff
@@ -3,7 +3,7 @@
 
 .home {
 	.landing {
-		height: 100vh;
+		height: calc(100vh + 101.2px); // height of navbar used to center the jeep
```

Comment:
Instead of messing with the landing height, could we instead add some padding-top to the entire Home page?

Reply:
Switched to using padding-top